.travis.yml

```diff
@@ -19,8 +19,7 @@ script:
   # - pyflakes: static analysis (syntax errors, unused variables, etc)
   # - pycodestyle: style checks
   # - mccabe: complexity analysis
-  - flake8 --exclude 'pyqtconsole/qt,build'
-  - flake8 --exclude '' pyqtconsole/qt --ignore F401,F403,F811
+  - flake8 --exclude build
```

Comment:
This can be simply `flake8` without arguments, also remove the `exclude = ...` line under `[flake8]` in `setup.cfg`. I think this would be best squashed into the commit that removes the `.qt` subpackage.

Reply:
Unfortunately I have not put the changes for the QtPy change in a separate branch to be merged. I have other changes between that one and my latest commit.

Reply:
It's fine. But in case you're interested, you can still easily squash using `git rebase -ir upstream/master` which allows reordering and squashing commits, succeeded by `git push -f`, which will update the pull request.

Reply:
I have cleaned up the string of commits to two single commits, one for QtPy and the other for the cursor thing and the blocking of input during longer commands (I always use time.sleep(10) for this)